### ISSUE TITLE
invalid escape sequence '\\[ in virt.prom

### ISSUE THREAD

**Issue by mbarbero** [openshift/node_exporter#82]
(First, please forgive me if I'm raising this issue in the wrong repository. Feel free to redirect me to the proper component if that's the case)

I have an alert in my OKD console for a node I recently added to my cluster. The alert says _Node Exporter text file collector failed to scrape_

I dug and the alert is raised by a pod on said node. The pod is from the node-exporter daemon set (hence the issue being raised here). Logs of the _node_exporter_ container in the pod say (every 3sec):

```
level=error ts=2021-02-09T16:21:34.807Z caller=textfile.go:209 collector=textfile msg="failed to collect textfile data" file=virt.prom err="failed to parse textfile data from \"/var/node_exporter/textfile/virt.prom\": text format parsing error in line 3: invalid escape sequence '\\['"
```

The file content is 

```
virt_platform{type="none",bios_vendor="IBM Corp.",bios_version="-\[D6E131CUS-1.05\]-",system_manufacturer="IBM",system_product_name="System x3550 M2 -\[7946AC1\]-",system_version="00",baseboard_manufacturer="IBM",baseboard_product_name="49Y6512     "} 1
```

Will it be fixed with https://github.com/openshift/node_exporter/commit/33de1ce2770cdebfdaf042cb7c4ae915061445c3 and https://github.com/openshift/node_exporter/commit/8f7995d77a4f913ef66c85674aaba133a1bc28e6?

Thanks!

System info:
Linux okdnode-5 5.9.16-200.fc33.x86_64 #1 SMP Mon Dec 21 14:08:22 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
OKD: 4.6.0-0.okd-2021-01-23-132511

**Comment by simonpasquier** [comment]
:wave: yes this is the same issue as described in https://bugzilla.redhat.com/show_bug.cgi?id=1917683 which is fixed by #78 and #79.

**Comment by mbarbero** [comment]
Thanks for the swift reply!
